frontend/src/hooks/useLogout.ts

```diff
@@ -1,14 +1,6 @@
-import { useContext } from 'react';
-import { AuthContext, AuthContextType } from '../context/AuthContext';
 import { auth } from '../firebase/config';
 
 const useLogout = () => {
-  const context = useContext(AuthContext) as AuthContextType;
-
-  if (!context) {
-    throw new Error('useLogout must be used within an AuthProvider');
-  }
-
   const logout = async () => {
     try {
       await auth.signOut();
```

Comment:
This method does not return any hook methods? Probably just make it as a function?